recipe/bld.bat

```diff
@@ -0,0 +1,34 @@
+mkdir build
+cd build
+
+cmake %SRC_DIR% -G "Ninja" ^
+      -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+      -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+      -DCMAKE_BUILD_TYPE="Release" ^
+      -DBUILD_PYTHON_BINDINGS=ON ^
+      -DBUILD_CLI_EXECUTABLES=ON ^
+      -DBUILD_SHARED_LIBS=OFF ^
+      -DBUILD_TESTS=OFF ^
+      -DMATLAB_BINDINGS=OFF ^
+      -DDOWNLOAD_STB_IMAGE=OFF ^
+      -DDOWNLOAD_ENSMALLEN=OFF ^
+      -DPYTHON_EXECUTABLE=%PYTHON% ^
+      -DUSE_OPENMP=ON ^
+      -DCMAKE_CXX_COMPILER=cl.exe ^
+      -DCMAKE_C_COMPILER=cl.exe ^
+      -DBoost_INCLUDE_DIRS=%LIBRARY_PREFIX%/include ^
+      -DBoost_USE_STATIC_LIBS=ON ^
+      -DBoost_USE_STATIC_RUNTIME=OFF ^
+      -DENSMALLEN_INCLUDE_DIR=%LIBRARY_PREFIX%/include ^
+      -DARMADILLO_INCLUDE_DIR=%LIBRARY_PREFIX%/include ^
+      -DARMADILLO_LIBRARIES="%LIBRARY_PREFIX%/lib/libarmadillo.lib" ^
+      -D_ARMA_SUPPORT_LIBRARIES="%LIBRARY_PREFIX%/lib/openblas.lib" ^
+      -DARMA_EXTRA_DEBUG=OFF
+      -DCMAKE_INSTALL_RPATH_USE_LINK_PATH=true ^
+      -DCOTIRE_MINIMUM_NUMBER_OF_TARGET_SOURCES=100000 ^
+      -DCMAKE_INSTALL_LIBDIR=lib ^
+      -DBoost_LIB_DIAGNOSTIC_DEFINITIONS=1 ^
```

Comment:
Looks like this didn't actually give any useful output, which is a bit disappointing.  I tried reading about the flag more... maybe try this:

```
-DCMAKE_CXX_FLAGS="-DBoost_LIB_DIAGNOSTIC_DEFINITIONS
```

Maybe that will give us more information.  Also, you could add

```
-DBoost_DEBUG=1
```

and that might help too.

Reply:
Ohh, I see, I dug further and the right incantation is actually

```
-DCMAKE_CXX_FLAGS="-DBOOST_LIB_DIAGNOSTIC"
```

(taken from FindBoost.cmake)

Reply:
Maybe for Windows you need to do `"/DBOOST_LIB_DIAGNOSTIC=1"` though, I'm not sure.

Reply:
`-DCMAKE_CXX_FLAGS="-DBOOST_LIB_DIAGNOSTIC"` I think it will work